Orange/widgets/evaluate/owliftcurve.py

```diff
@@ -61,49 +49,63 @@ class OWLiftCurve(widget.OWWidget):
                   "from the evaluation of classifiers."
     icon = "icons/LiftCurve.svg"
     priority = 1020
-    keywords = []
+    keywords = ["lift", "cumulative gain"]
 
     class Inputs:
-        evaluation_results = Input("Evaluation Results", Orange.evaluation.Results)
+        evaluation_results = Input(
+            "Evaluation Results", Orange.evaluation.Results)
+
+    class Warning(widget.OWWidget.Warning):
+        undefined_curves = Msg(
+            "Some curves are undefined; check models and data")
+
+    class Error(widget.OWWidget.Error):
+        undefined_curves = Msg(
+            "No defined curves; check models and data")
 
     settingsHandler = EvaluationResultsContextHandler()
     target_index = settings.ContextSetting(0)
     selected_classifiers = settings.ContextSetting([])
 
-    display_convex_hull = settings.Setting(False)
-    display_cost_func = settings.Setting(True)
-
-    fp_cost = settings.Setting(500)
-    fn_cost = settings.Setting(500)
-    target_prior = settings.Setting(50.0)
+    display_convex_hull = settings.Setting(True)
+    curve_type = settings.Setting(CurveTypes.LiftCurve)
 
     graph_name = "plot"
 
+    YLabels = ("Lift", "TPR")
```

Comment:
"TPR" used to be "TP Rate" and we still have "P Rate", so I suggest we use the same naming.

Reply:
This was unintentional. Fixed. Thanks for noticing.